roles/ceph-defaults/defaults/main.yml

```diff
@@ -522,13 +522,12 @@ rgw_zone: default
 #rgw_zone_user_display_name: "Zone User"
 #rgw_realm: milkyway # should be set by the user
 #rgw_multisite_proto: "http"
+#rgw_multisite_host: "{{ _radosgw_address if rgw_multisite_proto == 'http' else ansible_fqdn }}"
```

Comment:
why are you leaving this commented out?

Reply:
Since the variable shouldn't be set unless `rgw_multisite: true`